client/custom_flow.go

```diff
@@ -35,6 +35,19 @@ type CustomFlow struct {
 	IsTerragruntRunAll   bool             `json:"isTerragruntRunAll"`
 }
 
+type CustomFlowAssignmentScope string
+
+const (
+	CustomFlowOrganizationScope CustomFlowAssignmentScope = "ORGANIZATION"
```

Comment:
For now, you can remove the ability to configure org scope. keep the ability to add it in the future (When we will decide we support it)